src/widgets/__tests__/index.tests.ts

```diff
@@ -2,6 +2,14 @@ import { PlacesInstance } from 'places.js';
 import * as widgets from '../';
 import { Widget } from '../../types';
 
+/**
```

Comment:
@Haroenv WDYT?

Reply:
useful, although, I'm not sure this is the right place, what about adding "updating this file" and putting it in contributing.md?